docs/production-deployment/cloud/nexus/pricing.mdx

```diff
@@ -0,0 +1,22 @@
+---
+id: pricing
+slug: /cloud/nexus/pricing
+title: Pricing for Temporal Nexus
+description: Learn about the pricing structure for using Nexus. 
+sidebar_label: Temporal Nexus Pricing
+tags:
+  - temporal cloud
+  - nexus pricing
+keywords:
+  - Temporal Nexus pricing
+  - Temporal Cloud pricing
+---
+
+One Action to start or cancel a Nexus Operation in the caller Namespace.
+
+The underlying Temporal primitives such as Workflows, Activities, Signals created by a Nexus Operation handler (directly or indirectly) result in the normal Actions for those primitives.
+This includes retries for underlying Temporal primitives like Activities.
+
+No Action results for handling or retrying the Nexus Operation itself.
```

Comment:
we should make it clear here that while starting the retry of the operation itself is free, if the handler performs a billable action and then crashes (and would then be subsequently retried), that is charged. maybe it is already clear enough, but can see that being confusing potentially

Reply:
Added: "However, while the retry of the Nexus Operation incurs no charge, any billable action initiated by the handler (such as an Activity) will be charged if it fails and is subsequently retried."